gensrc/proto/olap_file.proto

```diff
@@ -72,190 +72,201 @@ message FooterPointerPB {
+  repeated uint32 rowsets_del = 5; // incremental optional
+  repeated uint32 deltas = 6;
+  optional CompactionInfoPB compaction = 7;
+  optional uint32 rowsetid_add = 8; // only available&useful in meta log
 }
 
 enum TabletMetaOpType {
-    OP_NONE = 0;
-    OP_ROWSET_COMMIT = 1;
-    OP_COMPACTION_COMMIT = 2;
-    OP_APPLY = 3;
+  OP_NONE = 0;
+  OP_ROWSET_COMMIT = 1;
+  OP_COMPACTION_COMMIT = 2;
+  OP_APPLY = 3;
 }
 
 message TabletMetaOpPB {
-    optional TabletMetaOpType type = 1;
-    optional EditVersionMetaPB commit = 2;
-    optional EditVersionPB apply = 3;
+  optional TabletMetaOpType type = 1;
+  optional EditVersionMetaPB commit = 2;
+  optional EditVersionPB apply = 3;
 }
 
 message TabletMetaLogPB {
-    repeated TabletMetaOpPB ops = 1;
+  repeated TabletMetaOpPB ops = 1;
 }
 
 message TabletUpdatesPB {
-    repeated EditVersionMetaPB versions = 1;
-    optional EditVersionPB apply_version = 2;
-    optional uint32 next_rowset_id = 3;
-    optional uint64 next_log_id = 4;
+  repeated EditVersionMetaPB versions = 1;
+  optional EditVersionPB apply_version = 2;
+  optional uint32 next_rowset_id = 3;
+  optional uint64 next_log_id = 4;
+}
+
+message BinlogConfigPB {
+  // Version of the configuration, and FE should deliver it to
+  // the BE when executing 'ALTER TABLE'. The configuration with
+  // a higher version can override that with a lower version.
+  optional int64 version = 1;
+  optional bool binlog_enable = 2;
+  optional int64 binlog_ttl_second = 3;
+  optional int64 binlog_max_size = 4;
 }
 
 message TabletMetaPB {
-    optional int64 table_id = 1;    // ?
-    optional int64 partition_id = 2;    // ?
-    optional int64 tablet_id = 3;   // OlapHeaderMessage.tablet_id
-    optional int32 schema_hash = 4; // OlapHeaderMessage.schema_hash
-    optional int32 shard_id = 5;    // OlapHeaderMessage.shard
-    optional int64 creation_time = 6;   // OlapHeaderMessage.creation_time
-    optional int64 cumulative_layer_point = 7;  // OlapHeaderMessage.cumulative_layer_point
-
-    optional TabletStatePB tablet_state = 8;
-    optional TabletSchemaPB schema = 9;
-    repeated RowsetMetaPB rs_metas = 10;
-    repeated RowsetMetaPB inc_rs_metas = 11;
-    optional AlterTabletPB alter_task = 12 [deprecated = true];
-    // if true, this tablet will not do compaction,
-    // and does not create init version
-    optional bool in_restore_mode = 13 [default = false];   // OlapHeaderMessage.in_restore_mode
-    // a uniqued id to identified tablet with same tablet_id and schema hash
-    optional PUniqueId tablet_uid = 14;
-    optional int64 end_rowset_id = 15;
-    optional RowsetTypePB preferred_rowset_type = 16 [default = BETA_ROWSET, deprecated = true];
-    optional TabletTypePB tablet_type = 17;
-    optional TabletUpdatesPB updates = 50; // used for new updatable tablet
-    optional bool enable_persistent_index = 51 [default = false]; // used for persistent index in primary index
+  optional int64 table_id = 1;    // ?
```

Comment:
Please do not change the ident in this PR, if it's necessary you could submit another PR

Reply:
it was changed unintentionally, have fixed it